spec/api/comment_thread_spec.rb

```diff
@@ -263,15 +263,15 @@ def move_to_front(ary, *vals)
             t5c.update(body: "changed!")
             t5c.save!
             actual_order = thread_result_order("activity", "desc")
-            expected_order = move_to_front(@default_order, "t5")
+            expected_order = @default_order
```

Comment:
Does this just show we are no longer sorting for update?  Do we have/need a test to show sorting by activity when it changes (i.e. for new responses)?